src/NUnitTestAdapterTests/NUnit.TestAdapter.Tests.csproj

```diff
@@ -4,7 +4,8 @@
   <PropertyGroup>
     <RootNamespace>NUnit.VisualStudio.TestAdapter.Tests</RootNamespace>
     <AssemblyName>NUnit.VisualStudio.TestAdapter.Tests</AssemblyName>
-    <TargetFrameworks>net46;netcoreapp1.0</TargetFrameworks>
+    <TargetFrameworks>net35;netcoreapp1.0</TargetFrameworks>
```

Comment:
@OsirisTerje Changing net46 to net35 is what is causing the NuGet restore failures. Any reason not to use `net46`?

Reply:
copy paste error when I reverted back from debugging this.  It is very annoying that VS don't really work with multiple targets, so I always remove the .netcore one.......,  just my way.....